timelock-agent/src/main/java/com/palantir/timelock/TimeLockStatus.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.timelock;
+
+public enum TimeLockStatus {
+    ONE_LEADER("There is exactly one leader in the Paxos cluster."),
+    NO_LEADER("There are no leaders in the Paxos cluster"),
+    MULTIPLE_LEADERS("There are multiple leaders in the Paxos cluster."),
```

Comment:
Is this intended to be "multiple nodes believe they're the leader", or "we really have multiple leaders"? (I'm guessing the first given usage in the ping check, but want to make sure)

Reply:
How about "Multiple nodes believe they are the leader"?